cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/TestCommand.java

```diff
@@ -204,6 +207,28 @@ public void execute() {
             return;
         }
 
+        if (this.rerunTests) {
+            // Cannot rerun failed tests using -a | -all flags
+            if (this.buildAll) {
+                CommandUtil.printError(this.errStream,
+                                       "Cannot specify --rerun-failed and -a | --all flags at the same time",
```

Comment:
Once the new package implementation is in place, we should allow running with the `-a` flag. Please create an issue to track this.

Reply:
Ack. Created issue to track this : https://github.com/ballerina-platform/ballerina-lang/issues/25382